examples/aditof-demo/aditofdemoview.cpp

```diff
@@ -844,8 +844,13 @@ void AdiTofDemoView::render() {
 
         if (valueFieldSelectedG) {
             if ((key >= 48 && key <= 57) || (key == 46)) {
-                valueG += pressedValidKey;
-                IRGamma = stof(valueG);
+                if ((valueG.size() < 11) &&
```

Comment:
Wouldn't it make sense to try to convert the newly formed string to float and if that fails, invalidate the key? And avoid all this complicated logic?

Reply:
modified. thanks